pom.xml

```diff
@@ -99,8 +100,13 @@
 				<artifactId>mockito-core</artifactId>
 				<version>${mockito.version}</version>
         <scope>test</scope>
-			</dependency>
-
+      </dependency>
+      
+      <dependency>
+        <groupId>com.beust</groupId>
+        <artifactId>jcommander</artifactId>
```

Comment:
JCommander seems to be a fine choice but given the many choices for command line argument parsing, I wonder was there a particular reason to pick this library, e.g., Apache Commons CLI? JCommander seems to be annotation based which is nice.

Reply:
let me try Apache Commons CLI  , seems a better alternative, hadnt checked it out. was more familiar with JCommander

Reply:
@bashir2 ,it seems Jcommander is the best alternative ,on addition of being annotation based